examples/complete/example.tf

```diff
@@ -1,13 +1,14 @@
 provider "azurerm" {
   features {}
+  subscription_id = "2334-12-343-23-##-34343"
```

Comment:
Can we use format fo subscription >> 000001-11111-1223-XXX-XXXXXXXXXXXX

Reply:
done